jerry-core/parser/js/js-parser-util.c

```diff
@@ -731,11 +731,11 @@ parser_error_to_string (parser_error_t error) /**< error code */
     }
     case PARSER_ERR_NUMBER_TOO_LONG:
     {
-      return "Number too long.";
+      return "Number is too long.";
```

Comment:
I know this is not correct grammatically but this kind of short form is frequently used, doesn't it?